src/use_factor_source/batch_use_factor_source_request.rs

```diff
@@ -0,0 +1,51 @@
+use crate::prelude::*;
+
+pub struct BatchUseFactorSourceRequest<ID, Path>
+where
+    ID: Hash,
+    Path: HasDerivationPath,
+{
+    /// Only signing supports skipping factor sources, key derivation does not,
+    /// so this closure is `None` for key derivation.
```

Comment:
(will update misleading comment about `closure` in coming PRs... used to be a closure)